src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,58 @@
+package duke;
+
+import command.Command;
+import exception.DukeException;
+import task.TaskList;
+import utility.UI;
+import utility.Parser;
+import utility.Storage;
+
+
+/**
+ * Main class for Duke
+ *
```

Comment:
Would deleting this empty line better aid being in line with the coding standard?